zerver/lib/hotspots.py

```diff
@@ -3,12 +3,11 @@
 from typing import Dict, List
 
 from django.conf import settings
-from django.utils.functional import Promise
 from django.utils.translation import gettext_lazy
 
 from zerver.models import UserHotspot, UserProfile
 
-INTRO_HOTSPOTS: Dict[str, Dict[str, Promise]] = {
+INTRO_HOTSPOTS: Dict[str, Dict[str, str]] = {
```

Comment:
`gettext_lazy` does not return `str`. This should be fixed in django-stubs.

Reply:
The changes noted have been undone for this PR. I will be working on a fix upstream.